data/windows.yaml

```diff
@@ -1563,6 +1563,19 @@ urls:
 - 'https://support.microsoft.com/en-us/kb/323525'
 - 'https://blog.malwarebytes.com/detections/pum-optional-disallowrun/'
 ---
+name: WindowsPortProxyConfiguration
+doc: Windows PortProxy registry keys (set by netsh portproxy command or manually).
+sources:
+- type: REGISTRY_KEY
+  attributes:
+    keys:
+    - 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PortProxy\v4tov4\tcp\*'
```

Comment:
Would it make sense to add support for `v4tov6`, `v6tov4`, and `v6tov6` sub keys as well ? And maybe different protocols such as `udp` that are suported by `netsh`?

Reply:
Yes, think that too to make the defintion more generic. No other keys/values are set there expect these port forwardings on my test machine.